README.md

```diff
@@ -62,24 +62,23 @@ In order to test a provider resource, there are three ways to do that.
 
 1. ### Using AWS CloudFormation website
 
-   After creating or updating or deleting a stack of a provider resource in AWS Cloudformation website, you need to check on [AWS Cloudformation](https://console.aws.amazon.com/cloudformation) for events which shows a status of a resource provider.
-   
-   To see logs, after uploading a stack, it creates automatically a log for [CloudWatch](https://console.aws.amazon.com/cloudwatch) in `Log groups`.
-   
+   After creating or updating or deleting a stack of a provider resource in AWS Cloudformation website, you need to check on [AWS CloudFormation](https://console.aws.amazon.com/cloudformation) for events which show the status of a resource provider.
```

Comment:
a slight adjustment for clarity:
After creating, updating or deleting a stack of a provider resource in the AWS CloudFormation console, you can check the [AWS CloudFormation Console ](https://console.aws.amazon.com/cloudformation) for events which will show the status of a resource provider.